src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -44,8 +44,9 @@ using namespace chip::app::Clusters::DoorLock;
 using namespace chip::app::Clusters::DoorLock::Attributes;
 using chip::Protocols::InteractionModel::Status;
 
-static constexpr uint8_t DOOR_LOCK_SCHEDULE_MAX_HOUR   = 23;
-static constexpr uint8_t DOOR_LOCK_SCHEDULE_MAX_MINUTE = 59;
+static constexpr uint8_t DOOR_LOCK_SCHEDULE_MAX_HOUR        = 23;
+static constexpr uint8_t DOOR_LOCK_SCHEDULE_MAX_MINUTE      = 59;
+static constexpr uint8_t DOOR_LOCK_ALIRO_CREDENTIAL_MIN_MAX = 65;
```

Comment:
```suggestion
static constexpr uint8_t DOOR_LOCK_ALIRO_CREDENTIAL_SIZE = 65;
```
I guess file style is SHOUTY_SNAKE_CASE, ok.

Reply:
🤣